### DISCUSSION TITLE
Report multiple packages

### DISCUSSION THREAD

**Discussion by mazipan** [npm/feedback#1013] [Registry]
Seems some keywords are being abused by people to hack seo.
It is hard to report one by one.

For instance: https://www.npmjs.com/search?q=judi%20slot

**Comment by alvinvin00** [comment]
same, searching for "[Judi Online](https://www.npmjs.com/search?q=judi%20online)" (lit. online gambling) also has a ton of results. Some a-hole thought it's a good idea to spam this junk on not just NPM, PyPI, Kaggle and even this very site is bombarded with this.

Please fix (or at least give us the tool needed) so we can avoid NPM from being blanket banned by our stupid ministry, as Google Docs, and Vercel was blanket banned days ago.

<img width="1280" alt="Screenshot 2023-10-10 104338" src="https://github.com/npm/feedback/assets/44573205/36db8379-0334-4458-b22f-f0ce1ff93847">
